packages/backend/migrations/20241125224212_create_tenants_table.js

```diff
@@ -6,8 +6,14 @@ exports.up = function (knex) {
   return knex.schema.createTable('tenants', function (table) {
     table.uuid('id').notNullable().primary()
     table.string('email').notNullable()
+    table.string('apiSecret').notNullable()
+    table.string('idpConsentUrl').notNullable()
+    table.string('idpSecret').notNullable()
```

Comment:
I think `idpConsentUrl` and `idpSecret` can be null, integrators may not actually be integrated with an IDP anyway. (echoing @tadejgolobic on our Tues call)